temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -273,6 +292,83 @@ public Builder setSslContext(SslContext sslContext) {
       return this;
     }
 
+    /**
+     * Sets default SSL context parameters that can be used with the temporal cloud service. Users
+     * that require additional customization may use {@link #setSslContext(SslContext)} directly.
+     *
+     * @param keyCertChainInputStream - an input stream for an X.509 certificate chain in PEM
+     *     format.
+     * @param keyPassword - the password of the keyFile, or null if it's not password-protected.
+     * @param keyInputStream - an input stream for a PKCS#8 private key in PEM format.
+     * @param fingerprints - a list of SHA1 fingerprints in hexadecimal form.
```

Comment:
can we give more detail on what these SHA1 fingerprints represent (does it represent acceptable Temporal Server certificate? Or acceptable CA that has issued Temporal Server Certificate?)

Reply:
This should be a fingerprint of the server's root CA. Updated the comment.